src/entities/trades/swapr-v3/SwaprV3.ts

```diff
@@ -196,22 +189,27 @@ export class SwaprV3Trade extends TradeWithSwapTransaction {
   public async swapTransaction(options: TradeOptions): Promise<UnsignedTransaction> {
     const isNativeIn = Currency.isNative(this.inputAmount.currency)
     const isNativeOut = Currency.isNative(this.outputAmount.currency)
-    invariant(!(isNativeIn && isNativeOut), 'SwaprV3Trade.swapTransaction: the router does not support both native in and out')
+    invariant(
+      !(isNativeIn && isNativeOut),
+      'SwaprV3Trade.swapTransaction: the router does not support both native in and out',
+    )
 
     const recipient = validateAndParseAddress(options.recipient)
     const amountIn = `0x${this.maximumAmountIn().raw.toString(16)}`
     const amountOut = `0x${this.minimumAmountOut().raw.toString(16)}`
 
     const isTradeExactInput = this.tradeType === TradeType.EXACT_INPUT
-    const routerContract = getRouterContract()
+
+    // Swapr Algebra Contract fee param is uint24 type
+    const algebraFee = parseFloat(this.fee.multiply(JSBI.BigInt(1_000_000)).toFixed(1))
```

Comment:
Let's turn this into a const, like 

`const ALGEBRA_FEE_PARTS_PER_MILLION = JSBI.BigInt(1_000_000)`

or just `PARTS_PER_MILLION` or `MILLION` or `_1_000_000` or `_1000000`

and use it here.

We have similar constants inside /constants folder, like `_10000`. Maybe we can create a new one there.

Reply:
Also, let's simplify this logic. I read `.toSignificant()` and `.toFixed()` code and we can use this:

```suggestion
    const algebraFee = this.fee.multiply(ALGEBRA_FEE_PARTS_PER_MILLION).toSignificant(1)
```

Reply:
I can turn it into a const, but I'll rather have it here next to where it's used.